x/swaprouter/types/constants.go

```diff
@@ -0,0 +1,6 @@
+package types
+
+const (
+	MinPoolAssets = 2
+	MaxPoolAssets = 8
```

Comment:
Should we specify this is for balancer pools only? Since its defined in swaprouter now

Reply:
I don't think this is balancer specific. Stableswap also utilizes these.

We can revisit this post-merge.